components/ShoppingItem.js

```diff
@@ -75,11 +108,55 @@ const StyledLink = styled(Link)`
   grid-area: details;
   border: 1px solid #362f23;
   border-radius: 15px;
-  height: 1.8rem;
-  width: 4.2rem;
+  height: 1.4rem;
+  width: 3.4rem;
   align-self: center;
   background-color: #fff4e9;
   text-decoration: none;
   color: #362f23;
   padding: 2px 0 0 6px;
+  font-size: 12px;
+`;
+
+const StyledButton = styled.button`
+  grid-area: delete;
+  border: none;
+  background-color: transparent;
+`;
+
+const DeleteIcon = styled(Delete)`
+  width: 20px;
+  height: 20px;
+  border-left: 1px dotted #362f23;
+`;
+
+const ConfirmDialog = styled.p`
+  grid-area: itemname;
+  align-items: center;
+  padding: 10px 0;
+  margin: 0;
```

Comment:
In your `styles.js` you have defined `margin: 0;` in the body, so I don't think it's necessary here. In our project we also selected `margin: 0;` and `padding: 0;` in the `*`-Selector in the `styles.js`, so that no automatic spacing settings can change anything anywhere, but we can choose all the spacing ourselves everywhere. Andrea gave me this tip once.